### ISSUE TITLE
There are some issues with the inliner library

### ISSUE THREAD

**Issue by kalenjordan** [magento-hackathon/Responsive-Email-Templates-with-CSS-Inliner#1]
Here's a side by side comparison of the same markup and CSS - one is using standalone CSS and the other is using the inliner.  It's mostly fine, but there are some conflicts.

For example, it doesn't give `td.wrapper` precedence over `table.column td` as it should in order to properly apply the padding to the social medium column of buttons.

![screen shot 2014-01-31 at 4 50 59 pm](https://f.cloud.github.com/assets/1542197/2056423/3d59033e-8adb-11e3-8428-66edb4fb9fb0.png)

**Comment by kalenjordan** [comment]
I tried using another library:
https://github.com/christiaan/InlineStyle

It seems quite a bit better, but there are still some places where it gets it wrong.  I'm beginning to get worried that if the CSS in-lining isn't bullet proof, there's kind of no point in using it.

Also, I'm pretty sure that you can't inline media queries.  So if we're not going to rely on inline css for media queries (which will probably be the really import CSS for nailing the responsive experience), then what's the point?

![screen shot 2014-01-31 at 5 18 21 pm](https://f.cloud.github.com/assets/1542197/2056511/d5b2d238-8ade-11e3-87b4-2b21a23f5776.png)

**Comment by nickweisser** [comment]
When using Ink's online inliner at http://zurb.com/ink/inliner.php the media queries are not inlined in the traditional sense, but are placed inside a <style> container just after the opening body tag. So our PHP inline class/library should do the same, then the media queries will work in all supported clients listed here http://zurb.com/ink/docs.php#compatibility

**Comment by kalenjordan** [comment]
Right, I tested that.  Both inliners seemed to ignore the media queries entirely instead of putting them in a `<style>` tag as they should.  I tried simply removing the media queries but there were still inconsistencies, so I don't think it's because of them.

**Comment by fplantinet** [comment]
Hello,
Try with https://github.com/jjriv/emogrifier
Result is better !

**Comment by kalenjordan** [comment]
Awesome thanks!

**Comment by erikhansen** [comment]
If any of you have experience with Emogrifier, it would be great to hear your experience with it. I'm looking for the best PHP Inliner library.

**Comment by kalenjordan** [comment]
Don't have any experience myself, @erikhansen - out of curiosity what are you using it for?

**Comment by erikhansen** [comment]
@kalenjordan I wish I could say. Ask me again after Imagine and I'll let you know.
